src/login/subscriptionTypes.ts

```diff
@@ -12,7 +12,7 @@ export interface ISubscriptionItem extends QuickPickItem {
 	subscription: AzureSubscription;
 }
 
-export interface ISubscriptionCache {
+export interface SubscriptionTenantCache {
```

Comment:
Soooo... we done with the I prefixes then? 😗

Reply:
😁😁  I decided to hop on the "no I" train but we should probs be consistent... Should we discuss next standup?

Reply:
lol no, I never want to have another 2 hour conversation about it again.  Let's just not use I moving forward